frontend/src/components/CreateCollectionModal/components/Content/index.tsx

```diff
@@ -137,26 +137,37 @@ const Content: FC<Props> = (props) => {
   if (isTombstonedCollection(data)) data = null;
   const { name, description, contact_email, contact_name } = data || {};
 
-  const [links, setLinks] = useState<Link[]>(
-    data?.links.map((link, index) => {
-      const {
-        link_name: linkName,
-        link_type: linkType,
-        link_url: linkUrl,
-      } = link;
-
-      // DOI links are specified as path only, all other links are specified as full URLs.
-      const url = isLinkTypeDOI(linkType) ? getDOIPath(linkUrl) : linkUrl;
-
-      return {
-        id: Date.now() + index,
-        isValid: true,
-        linkName,
-        linkType,
-        url,
-      };
-    }) || []
-  );
+  const [links, setLinks] = useState<Link[]>([]);
```

Comment:
would useMemo() work here? This way we get `links` one render cycle sooner each time!